Emby.Server.Implementations/Localization/Core/en-GB.json

```diff
@@ -76,7 +76,6 @@
     "StartupEmbyServerIsLoading": "Jellyfin Server is loading. Please try again shortly.",
     "SubtitleDownloadFailureForItem": "Subtitles failed to download for {0}",
     "SubtitleDownloadFailureFromForItem": "Subtitles failed to download from {0} for {1}",
-    "SubtitlesDownloadedForItem": "Subtitles downloaded for {0}",
     "Sync": "Sync",
     "System": "System",
     "TvShows": "TV Shows",
```

Comment:
Do the new keys need to be added here or any other languages, or do they fall back to en-US by default?

Reply:
@mark-monteiro Fall back to en-US afaik. I also added es because I know it, so I didn't see a reason to not do it.